pkgs/top-level/all-packages.nix

```diff
@@ -21461,9 +21461,9 @@ with pkgs;
 
   nvitop = callPackage ../tools/system/nvitop { };
 
-  nvtop = callPackage ../tools/system/nvtop { };
-  nvtop-nvidia = callPackage ../tools/system/nvtop { amd = false; };
-  nvtop-amd = callPackage ../tools/system/nvtop { nvidia = false; };
+  nvtop = callPackage ../tools/system/nvtop { cudaSupport = config.cudaSupport or true; };
+  nvtop-cuda = nvtop.override { cudaSupport = true; };
```

Comment:
I agree with proposed rename, but wonder is there any policy about renaming packages? Because it is a breaking change, albeit a small one. Shouldn't we add aliases with old names and some additional warnings? Or prepare an entry for a changelog like https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-22.05-incompatibilities ?

Reply:
Yes, please add aliases to `pkgs/top-level/aliases.nix`.

Reply:
IMHO, a release note entry isn't needed if aliases.nix is used. We should strive for automation and only add to release notes when there is no other way.

Reply:
I removed this change too.
There were too many things in my PR.

So I reverted all my changes to just upgrade nvtop to 3.0.1.